hack/deployer/runner/ocp.go

```diff
@@ -462,3 +462,29 @@ func (d *OCPDriver) baseDomain() string {
 	}
 	return baseDomain
 }
+
+func (d *OCPDriver) Cleanup(prefix string, olderThan time.Duration) ([]string, error) {
+	if err := d.authToGCP(); err != nil {
+		return nil, err
+	}
+	daysAgo := time.Now().Add(-olderThan)
+
+	params := d.bucketParams()
+	params["Date"] = daysAgo.Format(time.RFC3339)
+	params["E2EClusterNamePrefix"] = prefix
+	params["Region"] = d.plan.Ocp.Region
+
+	cmd := `gcloud compute instances list --verbosity error --zones={{.Region}}-a,{{.Region}}-b,{{.Region}}-c --filter="name~'^{{.E2EClusterNamePrefix}}-ocp.*-master.*' AND status=RUNNING" --format=json | jq -r --arg d "{{.Date}}" 'map(select(.creationTimestamp | . <= $d))|.[].name' | grep -o '{{.E2EClusterNamePrefix}}-ocp-[a-z]*-[0-9]*' | sort | uniq`
```

Comment:
Should we not list the zones in the region instead of hardcoding them? Maybe with something along those lines:
```bash
gcloud compute zones list --filter='region:https://www.googleapis.com/compute/v1/projects{{.GCloudProject}}regions/{{.Region}}' --format="value(selfLink.name())"
```

Reply:
I think this makes sense. I'll get that adjusted.